base_rest/__init__.py

```diff
@@ -0,0 +1,10 @@
+import logging
+from . import models
+from . import components
+from . import http
+
+logging.getLogger(__file__).warning(
```

Comment:
@lmignon it's now clear that we are going to support this for a longer period of time. I don't think keeping logging this has much value if not polluting logs.
Would you mind if we move this warning to the README?

Reply:
You're right. It's Ok for me.